Tests/ApolloCacheDependentTests/StarWarsServerTests.swift

```diff
@@ -3,9 +3,61 @@ import XCTest
 import ApolloTestSupport
 import StarWarsAPI
 
+
+protocol TestConfig {
+  func network() -> HTTPNetworkTransport
+}
+
+class DefaultConfig: TestConfig {
+  func network() -> HTTPNetworkTransport {
+    return HTTPNetworkTransport(url: URL(string: "http://localhost:8080/graphql")!)
+  }
+}
+
+class APQsConfig: TestConfig {
+  func network() -> HTTPNetworkTransport {
+    return HTTPNetworkTransport(url: URL(string: "http://localhost:8080/graphql")!,
+                                enableAutoPersistedQueries: true)
+  }
+}
+
+class APQsWithGetMethodConfig: TestConfig, HTTPNetworkTransportRetryDelegate{
+  func networkTransport(_ networkTransport: HTTPNetworkTransport, receivedError error: Error, for request: URLRequest, response: URLResponse?, retryHandler: @escaping (Bool) -> Void) {
+    retryHandler(true)
```

Comment:
Do you want to check that the received error is of the correct type? Otherwise this could lead to an infinite loop.